configure.ac

```diff
@@ -2624,6 +2624,9 @@ PPRINT_SUBTITLE([Modules])
 test "x$enable_module_dirauth" != "xno" && value=1 || value=0
 PPRINT_PROP_BOOL([Directory Authority (--disable-module-dirauth)], $value)
 
+test "x$enable_module_relay" != "xno" && value=1 || value=0
+PPRINT_PROP_BOOL([Relay (--disable-module-relay)], $value)
```

Comment:
This logic is the wrong way around: if the relay module is enabled, we should print "--enable-module-relay yes", but we print "--disable-module-relay yes".

A similar issue affects lots of different options in the configure summary.

Reply:
Yeah, the string is not suppose to reflect what was enabled. Maybe we should do that instead but that is much more configure.ac trickery.

Reply:
Not really, it's just s/disable/enable/g